test/distributed/org/apache/cassandra/distributed/impl/IsolatedExecutor.java

```diff
@@ -282,7 +282,7 @@ public static <T> T waitOn(Future<T> f)
         }
         catch (ExecutionException e)
         {
-            throw Throwables.throwAsUncheckedException(e.getCause());
+            throw new RuntimeException(e.getCause());
```

Comment:
I really should do this in OSS... I regret arguing for this when this was first done 😢 ... its more useful to know "where" in the test things failed, and less useful where in C*